CHANGELOG.md

```diff
@@ -1,20 +1,44 @@
 # Changelog
 
+## [0.29.0](https://github.com/plus3it/watchmaker/releases/tag/0.29.0)
+
+**Released**: 2025.01.21
+
+**Summary**:
+
+*   Adds support for Red Hat 9, CentOS Stream 9, Rocky Linux 9, Alma Linux 9, and
+    Oracle Linux 9
+*   Adds support for Windows Server 2022
+*   Removes support for Red Hat 7 and CentOS 7
+*   Removes suport for Windows Server 2016
+*   ash-linux-formula
+    -   Updated to support EL9 platforms
+*   ash-windows-formula
+    -   Updated to support Windows Server 2022 and Windows 11
+*   forescout-secure-connector-formula
+    -   Manages unit file for systemd service
+*   domain-join-formula
+    -   (Linux) Ensures FIPS crypto policies include AD suport on EL9
+*   scap-formula
+    -   Adds content for Windows 11 and Windows 2022
+    -   Adds mappings for EL9 platforms
+    -   Updates openscap content to 0.1.75
+
 ## [0.28.5](https://github.com/plus3it/watchmaker/releases/tag/0.28.5)
 
 **Released**: 2024.06.06
 
 **Summary**:
 
 *   Provides several new FAQs to address recent changes and SCAP findings
-*   Provides update on discontinuation of CentOS Stream 8 and possible work-around 
+*   Provides update on discontinuation of CentOS Stream 8 and possible work-around
 *   ash-linux-formula
-    -  Addresses findings related to UEFI-enabled systems
-    -  Removes remote log entry injected by scap content
+    -   Addresses findings related to UEFI-enabled systems
```

Comment:
Are the additional leading-spaced (after the bullet)  supposed to be here (and throughout the document)? The bullets being removed seem to not have them.

Reply:
There are no bullets being removed, that I'm seeing. Just some lint adjusting spaces.

The rest of this doc uses 3 spaces after the bullet marker, so was only updating things to match in the changelog for the prior release.